safedata_validator/resources.py

```diff
@@ -379,6 +388,57 @@ def _validate_ncbi(self) -> None:
             self.ncbi_database, "NCBI", ["nodes", "merge", "names"]
         )
 
+    def _validate_projects(self) -> None:
+        """Validate and load a project database.
+
+        This private function checks whether a project_database path: exists, is a CSV
+        file, and contains project data. It populates the instance ``project_id``
+        attribute.
+        """
+
+        if self.project_database is None:
+            LOGGER.info("No project database configured")
```

Comment:
Wonder about this message, it's very terse which is generally good, but I wonder if it risks giving the wrong impression, e.g. that not having a project database is something to worry about

Reply:
"Configuration does not use project IDs."?

Reply:
Sounds good